prime/infra/dev/prime-customer-api.yaml

```diff
@@ -422,8 +422,32 @@ paths:
           description: "Updated SIM profile for this user."
           schema:
             $ref: '#/definitions/SimProfile'
+        404:
+          description: "Sim Profile not found, or missing parameter."
+  "/regions/{regionCode}/simProfiles/{iccId}/installed":
+    put:
+      description: "Mark SIM Profile as installed by the app."
+      produces:
+        - application/json
+      operationId: "markSimProfileAsInstalled"
+      parameters:
+        - name: regionCode
+          in: path
+          description: "Region code"
+          required: true
+          type: string
+        - name: iccId
+          in: path
+          description: "ICC ID"
```

Comment:
It's "ICCID", not "ICC ID"

Reply:
Renamed description, but not the variable since it "may" bread API compatibility.